Python_env/hurrevac_settings.json

```diff
@@ -9,7 +9,10 @@
                     	"Exercise":0, 
                     	"Simulated":0},
 	"HurrevacStormsURL":"https://hvx.hurrevac.com/hvx-api/v1/advisories/storms", 
-	"HurrevacStormURL":"https://hvx.hurrevac.com/hvx-api/v1/advisories/storm",
```

Comment:
could you just add the "/" at the end of each one of these? If so, then you wouldn't have to concatenate it later on.

Reply:
Yes, this will make the code cleaner.